applications.json

```diff
@@ -7026,6 +7026,15 @@
             "languages": [
                 "swift"
             ]
+        },
+        {
+            "repo_url" : "https://github.com/jhspetersson/fselect",
+            "title" : "fselect",
+            "short_description" : "Command-line tool to search files with SQL syntax.",
+            "languages" : [
+                "Rust"
+            ],
+            "category" : "utilities"
```

Comment:
Hi @jhspetersson! Thanks for your contribution! 👍 
Unfortunately app object should contains all fields, it is required. Can you add fields like "icon_url",  "official_site", and other, and leave them empty. 
Also we support many categories for app, so please change  "category" to "categories".